integration-test/src/test/scala/com/cloudera/livy/test/JobApiIT.scala

```diff
@@ -157,16 +157,17 @@ class JobApiIT extends BaseIntegrationTestSuite with BeforeAndAfterAll with Logg
     assert(result === "hello")
   }
 
-  test("run scala jobs") {
+  test("run echo jobs") {
```

Comment:
Do you think it's a good idea to merge `minicluster-dependencies` with `test-lib`?